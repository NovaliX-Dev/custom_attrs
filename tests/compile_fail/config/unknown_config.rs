#![allow(unused)]
use custom_attrs::CustomAttrs;

#[derive(CustomAttrs)]
#[attr(
    /// Should pass

    #[unknown = "", another_unknown = ""]
    #[unknown2]
    
    pub a: usize
)]
enum Enum {
    #[attr(a = 5)]
    Variant1,

    #[attr(a = 3)]
    Variant2,
}

fn main() {
    // let a = Enum::Variant1.get_a();
}
