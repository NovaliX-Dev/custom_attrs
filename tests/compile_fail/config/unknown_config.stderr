error: Unknown config.
 --> tests/compile_fail/config/unknown_config.rs:8:7
  |
8 |     #[unknown = "", another_unknown = ""]
  |       ^^^^^^^

error: Unknown config.
 --> tests/compile_fail/config/unknown_config.rs:8:21
  |
8 |     #[unknown = "", another_unknown = ""]
  |                     ^^^^^^^^^^^^^^^

error: Unknown config.
 --> tests/compile_fail/config/unknown_config.rs:9:7
  |
9 |     #[unknown2]
  |       ^^^^^^^^
